src/eval/execution.rs

```diff
@@ -110,6 +123,16 @@ fn import_nix_module<'s>(
     ret.set(nix_fn.into());
 }
 
+fn debug_log<'s>(
+    scope: &mut HandleScope<'s>,
+    args: v8::FunctionCallbackArguments<'s>,
+    _ret: v8::ReturnValue,
+) {
+    // Log the first argument
+    let log_str = args.get(0).to_rust_string_lossy(scope);
+    println!("Log from JS: {}", log_str);
```

Comment:
Btw, you can also inline plain variables into the interpolation string: 
```suggestion
    eprintln!("Log from JS: {log_str}");
```